dat/models.py

```diff
@@ -0,0 +1,34 @@
+from pathlib import Path
+from typing import Dict, List, Optional
+
+from pydantic import BaseModel
+
+OUT_ROOT = Path('./out/reader_tests/generated/')
+
+
+class TableMetadata(BaseModel):
```

Comment:
`TableDescription`
`TableInfo`

Reply:
actually ... this has more to do with the test case, eh? maybe `TestCaseDescription` or `TestCaseInfo`